devops/templates/test-run-step-template.yml

```diff
@@ -17,8 +17,8 @@ parameters:
 - name: installationType
   type: string
 - name: pythonVersion
-  type: number
-  default: 3.7
+  type: string
+  default: "3.7"
```

Comment:
3.9 as default maybe? or even 3.10?

Reply:
Good point. Will adjust this and check for others!

Reply:
#966 is the only issue where this was discussed to my knowledge. You made the IMO correct point that we should remove 3.7 if the release is not in 2021 (which it isn't), so I'll remove all occurrences of 3.7 (and in pipelines increase to something higher).

Reply:
@fairlearn/fairlearn-maintainers FYI

Reply:
Would love to kick off the nightly builds to validate that it's working. [On that note, the "fixed" one is currently broken it seems.]